warehouse/query-core/src/main/java/datawave/query/tables/facets/MergedReadAhead.java

```diff
@@ -43,8 +43,8 @@ public MergedReadAhead(FacetedConfiguration facetedConfig, final Iterator<T> ite
         
         buf = QueueUtils.synchronizedQueue(new CircularFifoQueue(1));
         
-        startAndWait();
-        
+        startAsync();
+        awaitRunning();
```

Comment:
Does this need a try-catch, per your comment in the ScannerSession?

Reply:
I didn't see it thrown in any of the tests but I guess it would be better to have the try-catch to try and preserve the behavior.

Reply:
Thanks for the review but my branch was old and I forgot to rebase with latest before opening the PR.